package.json

```diff
@@ -426,6 +434,11 @@
                     },
                     "default": []
                   },
+                  "debuggerPath" : {
+                    "type" : "string",
+                    "description" : "The full path to the debugger",
```

Comment:
I would suggest adding to the end of this description 'on the target computer' (or similar if you don't like that description)